modules/jumpbox/jump-sg.tf

```diff
@@ -0,0 +1,59 @@
+#==============================================================
+# Jumpbox / jump-sg.tf
+#==============================================================
+
+# Security groups for Jumpbox
+
+resource "aws_security_group" "jump_ssh_sg" {
+  # Allow SSH to jump host
+  name = "jump_ssh"
+
+  ingress {
+    from_port   = 22
+    to_port     = 22
+    protocol    = "tcp"
+    cidr_blocks = ["${var.ssh_ip_address}"]
+  }
+
+  egress {
+    from_port   = 0
+    to_port     = 0
+    protocol    = "-1"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+
+  vpc_id = "${var.vpc_id}"
+
+  tags {
+    Name        = "${var.cluster_name}-${var.environment}-jump_ssh"
+    Owner       = "${var.owner}"
+    Cluster     = "${var.cluster_name}"
+    Environment = "${var.environment}"
+    Created_by  = "terraform"
+  }
+}
+
+resource "aws_security_group" "ssh_from_jump_sg" {
```

Comment:
This is a bit of legacy code that I'll be removing from the other examples soon, It's cleaner to just add the jumpbox rule (referencing the jumpbox security group) to the existing security group of a resource. This method requires a second security group for each resource, which can increase the chances of misconfiguring a security group.